devices/common_patches/dma/g0_add_dmamux_c7cr_c11cr.yaml

```diff
@@ -0,0 +1,437 @@
+DMAMUX:
+  _add:
+    C7CR:
+      displayName: C7CR
+      description: DMAMUX request line multiplexer channel 7 configuration register
+      addressOffset: 0x01C
+      size: 0x20
+      access: read-write
+      resetValue: 0x00000000
+      fields:
+        DMAREQ_ID:
+          description: DMA request identification Selects the input DMA request. See the DMAMUX table about assignments of multiplexer inputs to resources.
+          bitOffset: "0"
+          bitWidth: "6"
+          access: read-write
+        SOIE:
+          description: Synchronization overrun interrupt enable
+          bitOffset: "8"
+          bitWidth: "1"
+          access: read-write
+          enumeratedValues:
+            enumeratedValue:
+              B_0x0:
```

Comment:
`B_0x0` is useless name for variant.
Also we don't add enums in `common_patched`. enum related patches are in `peripherals`